lib/Dialect/Torch/Transforms/RefineTypes.cpp

```diff
@@ -624,23 +624,23 @@ ChangeResult TypeAnalyzer::visitOperation(
           AtenReluOp, AtenGeluOp, AtenCeilOp, AtenGeluBackwardOp,
           AtenBitwiseNotOp, AtenToPrimDeviceOp, AtenCpuOp, AtenContiguousOp,
           AtenFill_ScalarOp, AtenDetachOp, AtenMaskedFill_ScalarOp, AtenCopy_Op,
-          AtenCumsumOp, AtenLayerNormOp, AtenClampOp, AtenNegOp, AtenFloorOp,
```

Comment:
Same here.  This simply adds `AtenClampMaxOp` and `AtenClampMinOp` close to `AtenClampOp` to keep them all together.